src/dotnet-install.sh

```diff
@@ -321,11 +320,13 @@ get_normalized_architecture_from_architecture() {
     eval $invocation
 
     local architecture="$(to_lowercase "$1")"
+
+    if [[ $architecture == \<auto\> ]]; then
```

Comment:
minor: is case sensitivity a concern here to test for the auto case?  What if AuTO is found, for example?

Reply:
I wouldn't say so. `<auto>` is our internal default value for architecture. Users are not supposed to use it. If someone tries, script will take an early exit and print a proper error message.
https://github.com/dotnet/install-scripts/blob/a1a9c0c1513d72a8f1b70a693f2ca23e72820199/src/dotnet-install.sh#L1434
https://github.com/dotnet/install-scripts/blob/a1a9c0c1513d72a8f1b70a693f2ca23e72820199/src/dotnet-install.sh#L344

Reply:
Also, `architecture` is actually converted to lower case just above this check.